api/v1alpha1/project_config_types.go

```diff
@@ -20,6 +25,8 @@ type ProjectConfigSpec struct {
 	// PromotionPolicies defines policies governing the promotion of Freight to
 	// specific Stages within the Project.
 	PromotionPolicies []PromotionPolicy `json:"promotionPolicies,omitempty" protobuf:"bytes,1,rep,name=promotionPolicies"`
+	// ReceiverConfigs defines the default receivers for the project.
+	ReceiverConfigs []ReceiverConfig `json:"receivers,omitempty" protobuf:"bytes,2,rep,name=receivers"`
```

Comment:
I think for both the field name and the type, we should probably be more specific and call it _`WebhookReceiver`_. "Receiver" by itself doesn't tell me _what_ it receives and in this case, the comment isn't clarifying that much either. The more specific name will be immediately and effortlessly understood.